minefana-bungeecord/src/main/java/com/github/games647/minefana/collectors/BungeePlayerCollector.java

```diff
@@ -88,15 +100,15 @@ public void onServerSwitch(ServerSwitchEvent switchEvent) {
     }
 
     private String getCurrentServer(ProxiedPlayer player) {
-        String name;
+        String serverName;
         try {
-            name = player.getServer().getInfo().getName();
+            serverName = player.getServer().getInfo().getName();
```

Comment:
What error do you got error? Maybe it's better to check for the error condition and submit nothing on error instead of this error line or what do you think?

Reply:
Well after further checking player.getServer() is always null no matter what I do

Reply:
error looks like this https://pastebin.com/GkKi0haU